api/src/neo4j/queries/relatedReactions.js

```diff
@@ -67,6 +67,7 @@ CALL apoc.cypher.run("
   MATCH (:Reaction${m} {id: $rid})-[${v}]-(cm:CompartmentalizedMetabolite)
   WITH DISTINCT cm
   MATCH (cm)-[${v}]-(c:Compartment)-[${v}]-(cs:CompartmentState)
+  USING JOIN on c
```

Comment:
For the related-reactions, I do see a performance improvement. 
With `JOIN on`
```
time -p curl http://localhost:9080/api/v2/subsystems/transport_reactions/related-reactions?model=HumanGem > t1.txt
real: 45.01
```
without `JOIN on`

timeout after 60 seconds

Is that because neo4j is much slower on my Linux machine?

Reply:
For random components, could you try running it more times? The bottleneck is on compartments and it only gets noticeably slower if the randomized compartment is something like `Cytosol`.

Reply:
For related reactions, it was using a limit which is not hard-coded in the backend, so the request should look like `/api/v2/subsystems/transport_reactions/related-reactions?model=HumanGem&limit=200`.

Reply:
@nanjiangshu sorry, actually I notice that on the server the random components endpoint is not slow even if we are getting `Cytosol`, it is probably just my machine that was acting up the other day which was causing the long load times. I think the change still has a positive effect though, albeit much less significant.

Reply:
An improvement is an improvement :)